props/saber_BC_buttons.h

```diff
@@ -463,7 +489,12 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
       } else {
         thrust_begin_millis_ = millis();
       }
-  }
+
+    // Scroll Presets timer check - avoid beep/wav overlap
+    if (scroll_presets_timer_.timerCheck() && scroll_presets_) {
```

Comment:
I'm thinking that it might be cleaner to clear the "triggered" flag when scroll_presets_ becomes false rather than checking it here.

Reply:
Well, the timer is not ongoing while in the scroll presets mode, but rather just at the entry point to manage timing for the audio cue.
`triggered_` is set to false as soon as the timer expires.
The additional check for `&& scroll_presets` is just there for the off-chance the mode is exited super quickly and avoid playing font.wav unnecessarily.

Reply:
So anything else?